internal/assertions/stdout_assertion.go

```diff
@@ -40,7 +40,7 @@ func (a StdoutAssertion) Run(result executable.ExecutableResult, logger *logger.
 	if len(stdout) > len(a.ExpectedLines) {
 		logAllSuccessLogs(successLogs, logger)
 		logger.Errorf("! %s", stdout[len(a.ExpectedLines)])
-		return fmt.Errorf("Expected last stdout line to be %q, but found extra line: %q", a.ExpectedLines[len(a.ExpectedLines)-1], stdout[len(a.ExpectedLines)])
+		return fmt.Errorf("Expected nothing after the last stdout line of %q, but found extra line: %q", a.ExpectedLines[len(a.ExpectedLines)-1], stdout[len(a.ExpectedLines)])
```

Comment:
```suggestion
		return fmt.Errorf("Expected nothing after last stdout line (%q), found extra line: %q", a.ExpectedLines[len(a.ExpectedLines)-1], stdout[len(a.ExpectedLines)])
```

(bit shorter so that we aren't exceeding common line length limits)